components/Snackbar.vue

```diff
@@ -16,13 +16,26 @@ export default Vue.extend({
     };
   },
   created() {
+    // Empties potential already set buffer
+    this.displaySnackbar(
+      this.$store.state.snackbar.message,
+      this.$store.state.snackbar.color
+    );
     this.$store.subscribe((mutation, state) => {
       if (mutation.type === 'snackbar/SET_SNACKBAR_MESSAGE') {
-        this.message = state.snackbar.message;
-        this.color = state.snackbar.color;
-        this.model = true;
+        this.displaySnackbar(state.snackbar.message, state.snackbar.color);
       }
     });
+  },
+  methods: {
+    displaySnackbar(message: string, color: string) {
+      if (message !== '') {
+        this.message = message;
+        this.color = color;
+        this.model = true;
+        this.$store.commit('snackbar/RESET_MESSAGE');
```

Comment:
This should be triggered by an `action`.

Reply:
Done